Lib/_sitebuiltins.py

```diff
@@ -8,8 +8,14 @@
 # Note this means this module should also avoid keep things alive in its
 # globals.
 
+import os
 import sys
 
+sys.stdin = sys.__stdin__ = getattr(sys, '__stdin__', False) or os.fdopen(0, "r")
+sys.stdout = sys.__stdin__ = getattr(sys, '__stdout__', False) or os.fdopen(1, "w")
+sys.stderr = sys.__stdin__ = getattr(sys, '__stderr__', False) or os.fdopen(2, "w")
```

Comment:
`sys.__stdin__` is being set in all of these, rather than `sys.__std{in,out,err}__`